types/react-native/index.d.ts

```diff
@@ -6,6 +6,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//                 Ryan Thomas <https://github.com/RyanThomas73>
```

Comment:
I’m going to wait to hear back from @RyanThomas73 before merging this.

Reply:
@andy-ms Do you know if we can add a comment to this header that explicitly mentions what it means to add your name to this list? I’ve seen many people add their names only because they thought it was just a paper trail rather than wanting to sign-up for maintenance tasks.

Reply:
@timwangdev Thanks for the heads up. Removed my name from the definitions by list for now

Reply:
I’m actually wondering if the bot could pick it up and ask the question. The user could reply using the same thumbs up or thumbs down

Reply:
@alloy @iRoachie It would probably be worthwhile to update this part of the contributing section of `README.md` https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package to clarify that point; that was the reason I initially added my name.